src/icon.rs

```diff
@@ -261,6 +261,7 @@ impl Icons {
         m.insert("mp4", "\u{f03d}"); // ""
         m.insert("mustache", "\u{e60f}"); // ""
         m.insert("npmignore", "\u{e71e}"); // ""
+		m.insert("opus", "\u{f001}"); // ""
```

Comment:
Could you fix the spacing, after that we are good to go.

Reply:
Ah sorry, changed it to spaces now